ui/src/shared/components/TableGraph.tsx

```diff
@@ -426,31 +469,26 @@ class TableGraph extends Component<Props, State> {
     index: number
   }): number => {
     const {index} = column
-    const {
-      tableOptions: {fixFirstColumn},
-    } = this.props
+
     const {transformedData, columnWidths, totalColumnWidths} = this.state
-    const columnCount = _.get(transformedData, ['0', 'length'], 0)
     const columnLabel = transformedData[0][index]
 
-    let adjustedColumnSizerWidth = columnWidths[columnLabel]
+    if (this.fixFirstColumn && index === 0) {
+      return columnWidths[columnLabel]
+    }
 
-    const tableWidth = _.get(this, ['gridContainer', 'clientWidth'], 0)
-    if (tableWidth > totalColumnWidths) {
-      if (columnCount === 1) {
-        return columnSizerWidth
-      }
-      const difference = tableWidth - totalColumnWidths
-      const distributeOver =
-        fixFirstColumn && columnCount > 1 ? columnCount - 1 : columnCount
-      const increment = difference / distributeOver
-      adjustedColumnSizerWidth =
-        fixFirstColumn && index === 0
-          ? columnWidths[columnLabel]
-          : columnWidths[columnLabel] + increment
+    if (this.computedTableWidth <= totalColumnWidths) {
+      return columnWidths[columnLabel]
```

Comment:
consider combining this with the previous if statement since they return the same thing